pyproject.toml

```diff
@@ -0,0 +1,58 @@
+[project]
+name = "securedrop-builder"
+version = "0.1.0"
+requires-python = ">=3.9"
+
+[tool.ruff]
+line-length = 100
+select = [
+    # pycodestyle errors
+    "E",
+    # pyflakes
+    "F",
+    # isort
+    "I",
+    # flake8-gettext
+    "INT",
+    # flake8-pie
+    "PIE",
+    # pylint
+    "PL",
+    # flake8-pytest-style
+    "PT",
+    # flake8-pyi
+    "PYI",
+    # flake8-return
+    "RET",
+    # flake8-bandit
+    "S",
+    # flake8-simplify
+    "SIM",
+    # pyupgrade
+    "UP",
+    # pycodestyle warnings
+    "W",
+    # Unused noqa directive
+    "RUF100",
+]
+ignore = [
+    # Find contextlib.suppress() is harder to read
+    "SIM105",
+    # Find ternary statements harder to read
+    "SIM108",
+    # Flags any subprocess use
+    "S603",
+    # Ruff formatter makes a best effort, but may result in some lines exceeding 100 characters
+    "E501",
```

Comment:
How bad is it? Is it unreasonable to annotate the few exceptions with `noqa` or are there too many? (If it's that bad maybe we don't adopt ruff formatting just yet?)

I guess line length doesn't matter as a lint since if the formatting doesn't match it'll still fail CI, but it feels weird to me to set a max line length and then also ignore it.

Reply:
It was just one file. I've removed the exception and fixed up that one file; we just need to be aware that line length will sometimes require a bit more manual futzing than with `black`.

Reply:
Thanks, that seems fine with me, if it is problematic I think we'll only find out in much larger repos.